### PR TITLE
Pretty help printer

### PR DESCRIPTION
**Pull request by kjeom**
## Proposed changes

- make help print pretty
- added a category in each flag
- following flags have missed in FlagGroups
  - the flags are added in FlagGroups
  - although FlagGroups is not used anymore, I leave it as reference to make management easy.
```go
--common.docroot
--db.dynamo.read-only
--http-rpc.execution-timeout
--http-rpc.idle-timeout
--http-rpc.read-timeout
--http-rpc.write-timeout
--migration.dst.db.leveldb.cache-size
--ws-rpc.max-connections
--ws-rpc.max-subscription-per-conn
--ws-rpc.read-deadline
--ws-rpc.write-deadline 
```

## Types of changes

- [x] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by blukat29** [comment]
@kjeom Thanks! The new help looks pretty.

I have a question. The new help only shows the first alias of each flag. Previous help showed the name and aliases. Did you intentionally print only the first alias to encourage the alias?
```
ken help
  --datadir value, --common.datadir value, --migration.src.datadir value                     Data directory for the databases and keystore. This value is only used in local DB. (default: "/Users/krust/Library/KEN11") [$KLAYTN_DATADIR]

ken help
    --common.datadir value (default: "/Users/krust/Library/KEN")
          Data directory for the databases and keystore. This value is only used in local DB.
          EnvVar: [$KLAYTN_DATADIR]
```

**Comment by kjeom** [comment]
@blukat29 right. I just printed the first alias as default. If the alias is absent, the name will be printed.
I just printed only 1 because of just clean help test. we can discuss further.

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -280,640 +306,736 @@ var (
...
-		Usage:   "Target gas limit sets the artificial target gas floor for the blocks to mine",
-		Value:   params.GenesisGasLimit,
-		Aliases: []string{"common.target-gaslimit"},
-		EnvVars: []string{"KLAYTN_TARGETGASLIMIT"},
+		Name:     "targetgaslimit",
+		Usage:    "Target gas limit sets the artificial target gas floor for the blocks to mine",
+		Value:    params.GenesisGasLimit,
+		Aliases:  []string{"common.target-gaslimit"},
+		EnvVars:  []string{"KLAYTN_TARGETGASLIMIT"},
+		Category: "NETWORK",
```
Comment (ian0371) [inline cmd/utils/flags.go:653]:
```suggestion
		Category: "CONSENSUS",
```

**Comment by blukat29** [comment]
@kjeom
1. Feature-wise I think all names should be stated because our docs and scripts still use the original name. It would be too confusing if `--datadir` flag suddenly disappears. `geth help` does it too (e.g. `--header value, -H value`)
2. Aesthetics-wise it should be fine with multiple names. Since the new help is multiline, there's more room to list flag names.
3. Because the first alias is more organized, you might arrange them in the order of `Names[1] Names[0] Names[2..]`
	```
      # v1.10.2
	  --datadir value, --common.datadir value, --migration.src.datadir value                     Data directory for the databases and keystore. This value is only used in local DB. (default: "/Users/krust/Library/KEN11") [$KLAYTN_DATADIR]

	  # this PR
	    --common.datadir value (default: "/Users/krust/Library/KEN")
	          Data directory for the databases and keystore. This value is only used in local DB.
	          EnvVar: [$KLAYTN_DATADIR]

	  # suggestion
	    --common.datadir value, --datadir value, --migration.src.datadir value    (default: "/Users/krust/Library/KEN")
	          Data directory for the databases and keystore. This value is only used in local DB.
	          EnvVar: [$KLAYTN_DATADIR]
	```

File: cmd/utils/flags.go (hunk 1 above)
Comment (kjeom) [reply cmd/utils/flags.go:653]: This flag is actually accessed setupNetwork() function. I think that's why this flag has in NETWORK flagGroup.
Actually as you mentioned, It's not clear a bit. How do you think the refactoring these things as next step?

**Comment by kjeom** [comment]
@blukat29 I made it print all the names.

**Review by ian0371 — APPROVED** [review]

**Comment by blukat29** [comment]
@kjeom Please fix the failing tests in flaggroup_test.go

**Review by ian0371 — APPROVED** [review]

**Review by blukat29 — APPROVED** [review]

**Review by JayChoi1736 — APPROVED** [review]

**Review by nohkwak — APPROVED** [review]

**Pull request merged**